### DISCUSSION TITLE
Resident about text, non-english letters.

### DISCUSSION THREAD

**Discussion by SwineFeather** [TownyAdvanced/Towny#7569] [Q&A] [answered]
Like the title says, I would like to set `/res set about ` with non-english letters like åäö. How do I achieve that?
I have tried to change regex. But didn't work.` [Towny] Invalid string, resident about not set.`
  regex:
    name_filter_regex: '[\\\/]'
    name_check_regex: ^[\p{L}\*a-zA-Z0-9._\[\]-]*$
    string_check_regex: ^[a-zA-Z0-9 \s._\[\]\#\?\!\@\$\%\^\&\*\-\,\*\(\)\{\}åäöÅÄÖæøÆØñÑáÁéÉíÍóÓúÚýÝþÞðÐ]*$
    name_remove_regex: '[^\P{M}a-zA-Z0-9\&._\[\]-]'

**Comment by LlmDl** [comment, marked as answer]
Adding `\p{L}` into the string check regex should allow those characters.

**Comment by SwineFeather** [comment]
Thanks a lot. This worked!
